openstates/settings.py

```diff
@@ -1,6 +1,9 @@
 import os
 from .utils import transformers
 
+# settings for realtime flag
+S3_REALTIME_BASE = 's3://openstates-realtime-bills/'
```

Comment:
This probably should be configurable somehow...having a default is good, having it completely hard-coded is a problem later.